java-checks/src/test/files/checks/DoubleCheckedLockingCompliant.java

```diff
@@ -0,0 +1,52 @@
+class Compliant {
```

Comment:
is there a need for a separate file ? containing everything in the same file should be simpler (you can keep the inner class to help separate concerns)